apartments/forms.py

```diff
@@ -32,3 +32,46 @@ class Meta:
             'long_term',
             'immediate_entry'
         ]
+
+
+class ApartmentCreationForm(forms.ModelForm):
+
```

Comment:
1) For all the validations, please see: https://stackoverflow.com/questions/12384460/allow-only-positive-decimal-numbers
2) You are specifically looking for <0, what about 0 itself?